### ISSUE TITLE
Should respect "Virtual spaces" setting

### ISSUE THREAD

**Issue by TeaDrivenDev** [jedmao/tabsanity-vs#17]
When the "virtual spaces" setting is enabled, TabSanity should not change the caret position when it is already *after* the last character in the current line. Currently TabSanity and virtual spaces cannot be used at the same time.

**Comment by jedmao** [comment]
Totally agree, but I'm no longer supporting this plugin. I'm accepting PRs though if you want to contribute.

Outside of that, I use vscode now and I have a TabSanity plugin for that too that I'm more willing to update.

**Comment by TeaDrivenDev** [comment]
Yeah, I was just logging it for now and planning to look into solving it.

Thanks for mentioning the VS Code version; I was missing it there.
